src/Authorizer.ts

```diff
@@ -4,6 +4,7 @@ import * as casbin from 'casbin';
 import Permission from './Permission';
 import { StringKV } from './types';
 import * as Cache from './Cache';
+import { stringify } from 'ts-jest/dist/utils/json';
```

Comment:
why import this?

Reply:
Maybe forget to clear

Reply:
Sorry.This is my negligence. Deal with it immediately! @Zxilly @Sagilio